<launch>

    <arg name="rate" default="10.0"/>

    <node pkg="centroidal_planner" type="centauro_replay_node" name="centauro_replay_node" output="screen">
        <param name="is_model_floating_base" type="bool" value="true"/>
        <param name="model_type" value="RBDL"/>
        <param name="rate" value="$(arg rate)"/>
        <param name="log" type="bool" value="true"/>
    </node>

</launch>
